File: 2016/gara1/insegna/att/insegna.pas
function confronta(N: longint; var G, W: array of char): longint;
begin
    (* Mettete qui il codice della soluzione *)
    confronta := 42;
end;


const
    MAXN = 5000;
var
    N: longint;
    G, W: array[0..MAXN] of char;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    readln(fr, G);
    readln(fr, W);

    writeln(fw, confronta(N, G, W));
    close(fr);
    close(fw);
end.

File: ed2/gara1/insegna/att/insegna.c
#include <stdio.h>
#include <assert.h>

#define MAXN 5000

int confronta(int N, char* G, char* W) {
    // Mettete qui il codice della soluzione
    return 42;
}


char G[MAXN + 1], W[MAXN + 1];

int main() {
    FILE *fr, *fw;
    int N;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    assert(1 == fscanf(fr, "%s", G));
    assert(1 == fscanf(fr, "%s", W));

    fprintf(fw, "%d\n", confronta(N, G, W));
    fclose(fr);
    fclose(fw);
    return 0;
}
